src/js/models/tools/ellipse.js

```diff
@@ -47,7 +47,7 @@ define(function (require, exports, module) {
             backspaceKeyPolicy = new KeyboardEventPolicy(UI.policyAction.PROPAGATE_TO_BROWSER,
                 OS.eventKind.KEY_DOWN, null, OS.eventKeyCode.BACKSPACE);
         
-        Tool.call(this, "ellipse", "Ellipse", "ellipseTool", "tool.ellipse.select");
+        Tool.call(this, "ellipse", "Ellipse Tool", "ellipseTool", "tool.ellipse.select");
```

Comment:
As a matter of fact, they are the ones in the UI and are the ones being used by the search bar which is why I had to manually change them.
![](http://cdn2.gurl.com/wp-content/uploads/2013/07/tumblr_mdg3d5ij9z1qbtmhy.gif)